src/cache/manager.cpp

```diff
@@ -126,7 +126,7 @@ std::unique_ptr<T> ClientCacheWrapper<T>::query(
 		idx_resp = idx_con->write_and_read(ClientConnection::CMD_GET, req);
 	} catch ( const NetworkException &ne ) {
 		Log::error("Could not talk to index-server (%s:%d): %s", idx_host.c_str(), idx_port, ne.what());
-		throw OperatorException(ne.what());
+		std::throw_with_nested(OperatorException("", MappingExceptionType::SAME_AS_NESTED));
```

Comment:
would a constructor without message make sense here?

Reply:
I think so, will add it.